Mathlib/Topology/Algebra/InfiniteSum/Order.lean

```diff
@@ -403,7 +417,7 @@ requires more assumptions. -/
 @[positivity tsum _]
 def evalTsum : PositivityExt where eval {u α} zα pα e := do
   match e with
-  | ~q(@tsum _ $instCommMonoid $instTopSpace $ι $f) =>
+    | ~q(@tsumFilter _ $instCommMonoid $instTopSpace $ι $L $f ) =>
```

Comment:
I've never had to update tactics, can someone more experience check that this is ok?